book/src/chap20.asciidoc

```diff
@@ -282,22 +281,22 @@ julia> s = ones(String, 1, 3)
  ""  ""  ""
 ----
 
-The string unit element is an empty string.
+Uma string unitária é uma string vazia.
 
 [WARNING]
 ====
-+s+ is not a one-dimensional array:
++s+ não é um array unidimensional:
```

Comment:
Caso não for de fato uma array. Verificar com o @abelsiqueira.
```suggestion
+s+ não é um lista unidimensional:
```

Reply:
tudo é um array, mas aqui realmente não sei dizer se usamos "array" ou "lista", já que +s+ é uma matriz 1x3

Reply:
Acho que, como não foi definido o que é um "array" antes, talvez continuassemos a usar lista mesmo. Mas neste contexto, também é um array.

Reply:
@joaoluisro - Antes de aprovar, vou tomar esse ponto na reunião de amanhã então ;)

Reply:
@joaoluisro - Antes de aprovar, vou tomar esse ponto na reunião de amanhã então ;)